### PR TITLE
[REFACTOR] SpectrumSettings::

### PR DESCRIPTION
**Pull request by oliveralka**
peak data -> centroid
raw data -> profile

#3094

I tried to get the string representations as well - sometimes the naming is inconstant, or -imho- not fully understandable what was meant. Please point out if I forgot something or renamed too much.

### CODE REVIEW

File: src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h [hunk 1]
```diff
@@ -49,7 +49,7 @@ namespace OpenMS
   /**
     @brief This class represents the abstract base class of a signal to noise estimator.
 
-    A signal to noise estimator should provide the signal to noise ratio of all raw data points
+    A signal to noise estimator should provide the signal to noise ratio of all profile data points
```
Comment (timosachsenberg) [inline src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h]: check if this needs to be profile data

File: src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h [hunk 2]
```diff
@@ -106,7 +106,7 @@ namespace OpenMS
     {}
 
 
-    /// Set the start and endpoint of the raw data interval, for which signal to noise ratios will be estimated immediately
+    /// Set the start and endpoint of the profile data interval, for which signal to noise ratios will be estimated immediately
```
Comment (timosachsenberg) [inline src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h]: same here

File: src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h [hunk 3]
```diff
@@ -115,7 +115,7 @@ namespace OpenMS
       is_result_valid_ = true;
     }
 
-    /// Set the start and endpoint of the raw data interval, for which signal to noise ratios will be estimated immediately
+    /// Set the start and endpoint of the profile data interval, for which signal to noise ratios will be estimated immediately
```
Comment (timosachsenberg) [inline src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h]: and here

File: src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h [hunk 4]
```diff
@@ -207,9 +207,9 @@ namespace OpenMS
     /// stores the noise estimate for each peak
     std::map<PeakType, double, typename PeakType::PositionLess> stn_estimates_;
 
-    /// points to the first raw data point in the interval
+    /// points to the first profile data point in the interval
```
Comment (timosachsenberg) [inline src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h]: here

File: src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h [hunk 5]
```diff
@@ -207,9 +207,9 @@ namespace OpenMS
...
     PeakIterator first_;
-    /// points to the right position next to the last raw data point in the interval
+    /// points to the right position next to the last profile data point in the interval
```
Comment (timosachsenberg) [inline src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h]: here

File: src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/Fitter1D.h [hunk 6]
```diff
@@ -70,11 +70,11 @@ namespace OpenMS
     typedef Feature::CoordinateType CoordinateType;
     /// Quality of a feature
     typedef Feature::QualityType QualityType;
-    /// Raw data point type
```
Comment (timosachsenberg) [inline src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/Fitter1D.h]: peak type?

File: src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h (hunk 1 above)
Comment (oliveralka) [reply src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h]: @hroest - Is profile here OK?

**Comment by timosachsenberg** [comment]
looks good. let's see if it passes

File: src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h (hunk 1 above)
Comment (hroest) [reply src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h]: I think its ok but we also use it for chromatograms which are neither in centroid or profile mode ...

Comment (oliveralka) [reply src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h]: Then raw data seems to fit better in the case of SignalToNoiseEstimator - imho.

**Pull request merged**